build_runner/lib/src/build_script_generate/build_script_generate.dart

```diff
@@ -25,13 +26,18 @@ const scriptKernelCachedSuffix = '.cached';
 
 final _log = Logger('Entrypoint');
 
+final _lastShortFormatDartVersion = Version(3, 6, 0);
+
 Future<String> generateBuildScript() =>
     logTimedAsync(_log, 'Generating build script', _generateBuildScript);
 
 Future<String> _generateBuildScript() async {
   final info = await findBuildScriptOptions();
   final builders = info.builderApplications;
   final library = Library((b) => b.body.addAll([
+        Code(
+          '// @dart=${_lastShortFormatDartVersion.major}.${_lastShortFormatDartVersion.minor}',
+        ),
```

Comment:
This breaks all builds, the community tests just ran first :P It looks like `package:code_builder` does not insert a newline after the comment, so subsequent definitions got commented out. This fixes it:

```suggestion
        Code(
          '// @dart=${_lastShortFormatDartVersion.major}.${_lastShortFormatDartVersion.minor}\n',
        ),
```